docs/architecture/key-concepts.md

```diff
@@ -2,11 +2,19 @@
 
 ## Blocks
 
-Blocks are an abstract unit for organizing and composing content, strung together to create content for a webpage.
+Blocks are an abstract unit for structuring and interacting with content. When composed together they create the content for a webpage. Everything from a paragraph, to a video, to the site title is represented as a block.
 
-Blocks are hierarchical in that a block can be a child of or parent to another block. For example, a two-column Columns block can be the parent block to multiple child blocks in each of its columns.
+Blocks come in many different forms but also provide a consistent interface. They can be inserted, moved, reordered, copied, duplicated, transformed, deleted, dragged, and combined. Blocks can also be reused, allowing them to be shared across posts and post types and/or used multiple times in the same post. If it helps, you can think of blocks as a more graceful shortcode, with rich formatting tools for users to compose content.
 
-If it helps, you can think of blocks as a more graceful shortcode, with rich formatting tools for users to compose content. To this point, there is a new Block Grammar. Distilled, the block grammar is an HTML comment, either a self-closing tag or with a beginning tag and ending tag. In the main tag, depending on the block type and user customizations, there can be a JSON object. This raw form of the block is referred to as serialized.
+The settings and content of a block can be customized in three main places: the block canvas, the block toolbar, and the block inspector. Blocks can be limited or locked-in-place by _Templates_ and custom code.
```

Comment:
What does it mean for a block to be _limited_ in this context?

Reply:
I removed that part because it's not super clear and should be part of a broader topic on templates